Scripts/logging.py

```diff
@@ -190,6 +219,26 @@ def print_prepared_comments(current, scanVideoID, comments, j, loggingEnabled, s
     i += 1
     j += 1
 
+  # Re-order indexes
```

Comment:
I ended up re-ordering the indexes after all samples are added. Wasn't sure if the list was supposed to be in order or if I broke something in the process.